docs/architecture/NONCLOUD.md

```diff
@@ -0,0 +1,38 @@
+# Non-cloud architecture
+
+> [!NOTE]
+> This doc is meant to be purely informative, and makes no commitment
+> to any support level nor future plan.
+
+At the moment, TesseraCT only supports [Google Cloud Platform](https://cloud.google.com)
+(GCP) and [Amazon Web Services](https://aws.amazon.com) (AWS) deployments,
+using platform-specific products, such as Spanner with GCS on GCP and RDS with
+S3 on AWS.
+
+TesseraCT is based on [Tessera](https://github.com/transparency-dev/tessera),
+which itself offers two implementations that can be deployed outside of GCP or AWS:
+
+- [POSIX](https://github.com/transparency-dev/tessera/tree/main/storage/posix)
+- [MySQL](https://github.com/transparency-dev/tessera/tree/main/storage/mysql)
+
+While Tessera's [AWS implementation](https://github.com/transparency-dev/tessera/tree/main/storage/aws)
+uses [RDS](https://aws.amazon.com/rds/) and [S3](https://aws.amazon.com/s3/),
+its configuration APIs _should_ be compatible with any MySQL database, and S3
+compatible backend, such as [MinIO](https://min.io/). If you'd like to run
+TesseraCT outside of GCP or AWS, we believe that such a setup would be easier to
+implement and to run at the scale of CT rather than using Tessera's POSIX or
+MySQL implementations (no need to implement a read server, write and read
+resources fully decoupled, etc.).
```

Comment:
I'm not sure I agree with this statement :)

We know, for example, that there is a POSIX-based sunlight instance already up and running. I'm also not sure that anyone on the team has much real-world experience operating their own minio instance at scale.